chroma-common/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../include/Makefile.version
```

Comment:
@tanabarr: Sorry, I was not clear.  The patch I have it not about extracting chroma-common (or any other ingrained bits) out of overlapping modules but rather about extracting the various already defined packages such as chroma-manager, chroma-agent, chroma-diagnostics, etc. out of a single "project" into distinct projects.

The general guideline we should be moving toward is one RPM specfile per project rather than the current 3-4 that we have in the "chroma" project.